pandas/core/frame.py

```diff
@@ -4818,66 +4818,82 @@ def aggregate(self, func, axis=0, *args, **kwargs):
 
     def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
               result_type=None, args=(), **kwds):
-        """Applies function along an axis of the DataFrame.
+        """
+        Apply a function along an axis of the `Series`.
```

Comment:
\`Series\` -> DataFrame

Reply:
Ai, we were all doing the review at the same time :-)

Reply:
Yep. Sorry guys :)

Reply:
@TomAugspurger I took into account your suggestion in the commit c593a70